modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -108,57 +108,30 @@ def config
     supplemental_claims: 'Supplemental Claims'
   }.freeze
 
-  DEFAULT_READ_SCOPE_DISPLAY_NAMES = {
-    'veteran/appeals.read': 'Appeals info',
-    'representative/appeals.read': 'Appeals info',
-    'system/appeals.read': 'Appeals info'
-  }.freeze
-
-  DEFAULT_WRITE_SCOPE_DISPLAY_NAMES = {
-    'veteran/appeals.write': 'Ability to submit appeals',
-    'representative/appeals.write': 'Ability to submit appeals',
-    'system/appeals.write': 'Ability to submit appeals'
-  }.freeze
+  DEFAULT_READ_SCOPE = { 'appeals.read': 'Appeals info' }.freeze
+  DEFAULT_WRITE_SCOPE = { 'appeals.write': 'Ability to submit appeals' }.freeze
 
-  OAUTH_SCOPE_DISPLAY_NAMES = {
+  OAUTH_SCOPES = {
     appeals_status: {
-      'veteran/AppealsStatus.read': 'Status of appeals and decision reviews',
-      'representative/AppealsStatus.read': 'Status of appeals and decision reviews',
-      'system/AppealsStatus.read': 'Status of appeals and decision reviews'
+      'AppealsStatus.read': 'Status of appeals and decision reviews'
```

Comment:
The `veteran`/`representative`/`system` prefix is now added later, so that `veteran` and `representative` scopes can be provided for the Authorization Code Grant flow and `system` scopes can be provided for CCG.